### PR TITLE
Unification api

### PR DESCRIPTION
**Pull request by FrancescoXX**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,8 +10,15 @@
...
 
-app.config.from_pyfile('./config/appconfig.cfg')
-CONF = f"postgresql://{app.config['PG_USER']}:{app.config['PG_PASSWORD']}@{app.config['PG_HOST']}:{app.config['PG_PORT']}/{app.config ['PG_DATABASE']}"
+# app.config.from_pyfile('./config/appconfig.cfg')
+
+pg_user = os.environ.get("PG_USERNAME")
+pg_password = os.environ.get("PG_PASSWORD")
+pg_host = os.environ.get("PG_HOST")
+pg_port = os.environ.get("PG_PORT")
+pg_database = os.environ.get("PG_DATABASE")
```
Comment (zsimjee) [inline app.py:19]: can we add defaults to these in case they're not defined? like localhost, 5432, postgres, etc

File: app.py [hunk 2]
```diff
@@ -90,19 +97,21 @@ def update_item():
 def delete_item():
     return "delete_item"
 
-@app.route('/item', methods=['PUT'])
+# Get All items
+@app.route('/local-item', methods=['GET'])
```
Comment (zsimjee) [inline app.py:101]: is there still an API to get a single item?

File: app.py (hunk 1 above)
Comment (FrancescoXX) [reply app.py:19]: yes sure we can. what should the value be?

Comment (zsimjee) [reply app.py:19]: username: postgres
password: undefined
host: localhost
port: 5432
database: postgres

**Review by zsimjee — APPROVED** [review]

**Pull request merged**